src/autocomplete/CommandProvider.tsx

```diff
@@ -58,7 +63,7 @@ export default class CommandProvider extends AutocompleteProvider {
                 matches = Commands;
             } else {
                 // otherwise fuzzy match against all of the fields
-                matches = this.matcher.match(command[1]);
+                matches = this.matcher.match(command[1], limit);
```

Comment:
would be good to get a comment on L63 saying that we explicitly ignore the limit there